packages/vuetify/package.json

```diff
@@ -62,7 +62,10 @@
       "default": "./lib/styles/main.css"
     },
     "./styles/*": "./lib/styles/*",
-    "./framework": "./lib/framework.mjs",
+    "./framework": {
```

Comment:
vuetify/framework is deprecated, it's the same as vuetify now

Reply:
ok, closing the PR